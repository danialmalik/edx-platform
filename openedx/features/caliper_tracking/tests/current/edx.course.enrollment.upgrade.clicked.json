{
  "accept_language": "en-US,en;q=0.9,ur-PK;q=0.8,ur;q=0.7",
  "agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36",
  "context": {
    "course_id": "",
    "org_id": "",
    "path": "/event",
    "user_id": 7
  },
  "event": "{\"location\": \"learner_dashboard\"}",
  "event_source": "browser",
  "event_type": "edx.course.enrollment.upgrade.clicked",
  "host": "tasawernawaz.sandbox.edx.org",
  "ip": "116.58.62.58",
  "name": "edx.course.enrollment.upgrade.clicked",
  "page": "https://tasawernawaz.sandbox.edx.org/dashboard",
  "referer": "https://tasawernawaz.sandbox.edx.org/dashboard",
  "session": "f3029807d5ac0b71b59b53e9e789f0fc",
  "time": "2018-10-17T13:08:35.005410+00:00",
  "username": "honor"
}