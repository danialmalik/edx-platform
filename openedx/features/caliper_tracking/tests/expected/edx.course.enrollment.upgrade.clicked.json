{
  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
  "action": "NavigatedTo",
  "actor": {
    "id": "edx.devstack.lms:18000/u/honor",
    "name": "honor",
    "type": "Person"
  },
  "eventTime": "2018-10-17T13:08:35.005Z",
  "extensions": {
    "extra_fields": {
      "accept_language": "en-US,en;q=0.9,ur-PK;q=0.8,ur;q=0.7",
      "agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36",
      "course_id": "",
      "event": "{\"location\": \"learner_dashboard\"}",
      "event_source": "browser",
      "event_type": "edx.course.enrollment.upgrade.clicked",
      "host": "tasawernawaz.sandbox.edx.org",
      "ip": "116.58.62.58",
      "org_id": "",
      "page": "https://tasawernawaz.sandbox.edx.org/dashboard",
      "path": "/event",
      "session": "f3029807d5ac0b71b59b53e9e789f0fc",
      "user_id": 7
    }
  },
  "id": "urn:uuid:13bc41f6-8406-4006-b929-c9f1ac156ada",
  "object": {
    "id": "http://edx.devstack.lms:18000/dashboard",
    "type": "WebPage"
  },
  "referrer": "https://tasawernawaz.sandbox.edx.org/dashboard",
  "type": "NavigationEvent"
}